.github/workflows/lint.yaml

```diff
@@ -1,13 +1,7 @@
 ---
 name: 'Lint: YAML'
 
-on:
-  workflow_dispatch:
```

Comment:
Make sure you keep `workflow_dispatch`, this will allow us to manually execute these workflows on arbitrary commits